app/presenters/compute_resource_presenter.rb

```diff
@@ -0,0 +1,93 @@
+class ComputeResourcePresenter
+  def self.for_cr(compute_resource, view)
```

Comment:
Can we name the method `for`? That makes the API a bit nicer. Also, I'd make `view` a kwarg. `ComputeResourcePresenter.for(compute_resource, view: view)`